packages/frontend/src/app/layout.tsx

```diff
@@ -22,6 +23,7 @@ export default function RootLayout({
 }: Readonly<{ children: React.ReactNode }>) {
   return (
     <html lang="en">
+      <Navbar/>
       <body>{children}</body>
```

Comment:
This causes a hydration error with Next.js, moving it to the body seems to work fine

```suggestion
      <body>
          <Navbar/>
          {children}
      </body>
```